Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/README_MixedRealityKeyboard.md

```diff
@@ -0,0 +1,19 @@
+# Mixed Reality and HoloLens Keyboard helpers
+
+MRTK provides several experimental helper components to assist with launching and reading text from the [System Keyboard](../../../../../Documentation/README_SystemKeyboard.md). 
+
```

Comment:
It might be helpful to note that the keyboard does not work in editor or while remoting.

Reply:
Done.